api/v1beta1/agentserviceconfig_types.go

```diff
@@ -88,6 +88,12 @@ type AgentServiceConfigSpec struct {
 	// that are used if one the operators fails to be successfully deployed
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Must-Gather Images"
 	MustGatherImages []MustGatherImage `json:"mustGatherImages,omitempty"`
+
+	// IPXEHTTPRoute is controlling whether the operator is creating plain HTTP routes
```

Comment:
I think a comment from @mhrivnak about documenting which endpoints are exposed over http got lost here.

Reply:
Updated the comment